### ISSUE TITLE
3D representation for labelmap

### ISSUE THREAD

**Issue by floryst** [KitwareMedical/paraview-medical#15]
Ideally this will be a polygonal representation, as mixed volumes in vtk.js is not yet supported.

**Comment by sankhesh** [comment]
If you're going for segmentation based on the label map, polygonal representation makes sense. IMO, just for displaying the label map, it would be better to have a volumetric representation.

**Comment by sankhesh** [comment]
For volume rendering, I suggest an alternate approach that allows you to specify material properties and transfer functions per label. This method would support all the different blend modes without requiring special cases for multi-component handling as described above.

**Comment by PaulHax** [comment]
>  i suggest an alternate approach that allows you to specify material properties and transfer functions per label.

Sounds nice.  Would you do that with this PR's "additive" preset?
- https://github.com/Kitware/vtk-js/pull/3029#discussion_r1514929706

**Comment by sankhesh** [comment]
Nope. Ideally, there will be a separate label volume that would give each voxel a label and the mapper would then index into the label map to figure out the current sample color and alpha. This approach could be used in addition to whatever blending function is employed for multiple components.
